LICENSE.md

```diff
@@ -1,6 +1,6 @@
 # MIT License
 
-Copyright © 2022 Bentley Systems, Incorporated. All rights reserved.
+Copyright © 2023 Bentley Systems, Incorporated. All rights reserved.
```

Comment:
sorry it should be 2022-2023; for all the licenses

Reply:
[like this](https://github.com/iTwin/object-storage/pull/72/commits/78b54c30a7dd8e8de4796327ea21d40355cc8671)?